core/src/main/java/org/apache/accumulo/core/util/compaction/CompactionServicesConfig.java

```diff
@@ -50,16 +50,32 @@ private long getDefaultThroughput() {
         .getMemoryAsBytes(Property.TSERV_COMPACTION_SERVICE_DEFAULT_RATE_LIMIT.getDefaultValue());
   }
 
+  @SuppressWarnings("deprecation")
   private Map<String,String> getConfiguration(AccumuloConfiguration aconf) {
-    return aconf.getAllPropertiesWithPrefix(Property.TSERV_COMPACTION_SERVICE_PREFIX);
+    Map<String,String> properties = new HashMap<>();
+    properties.putAll(aconf.getAllPropertiesWithPrefix(Property.TSERV_COMPACTION_SERVICE_PREFIX));
+    properties.putAll(aconf.getAllPropertiesWithPrefix(Property.COMPACTION_SERVICE_PREFIX));
+    // Return unmodifiable map
+    return Map.copyOf(properties);
   }
 
+  @SuppressWarnings("deprecation")
   public CompactionServicesConfig(AccumuloConfiguration aconf) {
     Map<String,String> configs = getConfiguration(aconf);
 
+    String oldPrefix = Property.TSERV_COMPACTION_SERVICE_PREFIX.getKey();
+    String newPrefix = Property.COMPACTION_SERVICE_PREFIX.getKey();
```

Comment:
Seems like this could mix config for a compaction service from both prefixes, which seems confusing.  What do you think about the following behavior?

 * If a compaction service is defined in both only use properties with the new prefix.  Log something about the old prefix props being ignored.
 * When a compaction service is only defined using the old prefix, then log something about it being deprecated and use the props.
 * When a compaction service is only defined using the new prefix, then use it w/o logging anything.

Reply:
Added behavior to check which prefix is used by the planner definition and then allow old properties to be set with logged statement and new properties can be defined with no logging messages.

Reply:
For this to work correctly, I did have to remove the old properties defined for the`meta`,root`, and `default` compaction services. 

I could remove the new properties and keep the old ones until the elasticity branch if that's the better way of handling things.

Reply:
After discussion with @keith-turner, the new property examples were removed so that property collision did not occur. 

The new property behavior is as follows: 

1. If a compaction service property is set via the default configuration, and it uses the old prefix, then the property will be used and no log messages will be generated. 

2. If compaction service properties are set by the user, and all properties for that service use the old prefix, then the properties will be used and a warning message will be logged, informing the user to update the property to the new prefix.

3. If a single compaction service property is set using the new prefix, then all properties for that service must use the new prefix. Compaction service properties using the old prefix will be dropped and a warning message will be logged. 

4. If all properties for a compaction service use the new prefix, then nothing is logged and the compaction service properties are used.